plugins/catalog-backend/src/service/CatalogBuilder.ts

```diff
@@ -685,17 +685,17 @@ export class CatalogBuilder {
       }),
     ];
 
-    const builtinKindsEntityProcessor = new BuiltinKindsEntityProcessor();
+    const systemEntityModelProcessor = new SystemEntityModelProcessor();
```

Comment:
Just to confirm my understanding, this is the reason why we don't need to install this module manually as you mentioned in the pull request description, right? If someone wants to replace the default entity processor, what do they need to do? Would the procedure be to install a custom module that returns the same processor name as this one? It might be a good idea to mention this procedure in the module documentation.

Reply:
_For now_, that's how things have worked even before this PR. If you want to entirely replace the builtin model, you need to make a processor with that name, which replaces the builtin one.

This is clearly very opaque and just an unfortunate effect of the fact that this model was so hard-coded from the start. In the future, we should make a very explicit entity model handling (not using processors at all) where you do like `catalogModelExtensionPoint.setEntityModel(myModel)` instead. So this is all historic compatibility cruft for the time being